site/content/docs/user/known-issues.md

```diff
@@ -389,37 +375,46 @@ docker: Error response from daemon: open /dev/dma_heap: permission denied.
 
 Although the policy has been fixed in Fedora 34, the fix has not been backported to Fedora 33 as of June 28, 2021. Putting SELinux in permissive mode (`setenforce 0`) is one known workaround. This disables SELinux until the next boot. For more details, see [kind#2296].
 
-## Failure to Create Cluster with Docker Desktop as Container Runtime
+## Docker Desktop for macOS and Windows
+
+Docker containers cannot be executed natively on macOS and Windows, therefore Docker Desktop runs them in a Linux VM. As a consequence, the container networks are not exposed to the host and you cannot reach the kind nodes via IP. You can work around this limitation by configuring [extra port mappings](https://kind.sigs.k8s.io/docs/user/configuration/#extra-port-mappings) though.
```

Comment:
This isn't the only solution and we should elaborate more if we're going to mention it here, they have a number of options including the recent work in cloud provider kind, proxies, running tests inside the cluster network, etc.

Maybe at least "You can work around this limitation by, for example, configuring [extra port mappings] amongst other options"

Reply:
This is showing up as new in the diff, but this is an existing section: https://kind.sigs.k8s.io/docs/user/known-issues/#docker-desktop-for-macos-and-windows

I'll tweak the wording a bit to try to improve it though.